CHANGES.md

```diff
@@ -30,9 +30,7 @@
 
 ### Database migration
 
-- Optional database schema migration, which can be run via `dagster instance migrate`:
-    - Improves Dagit performance by adding database indexes which should speed up the run view as well as a range of asset-based queries.
-    - Enables multi-dimensional asset partitions and asset versioning.
+- This release requires a database schema migration, which can be run via `dagster instance migrate`.
```

Comment:
may want to still leave the reason why this is necessary.

Reply:
👍